test/integration/features/story_health.feature

```diff
@@ -54,6 +63,7 @@ Feature:
         And executing "oc expose svc/httpd-ex" succeeds
         And with up to "2" retries with wait period of "60s" http response from "http://httpd-ex-testproj.apps-crc.testing" should have status code "200"
 
+    @linux @darwin @windows
```

Comment:
Do we need to be explicit about all the platforms here? Or would not specifying anything mean that this should always be used?

Reply:
I'll check that. If it is the case, we can remove it as a part of another PR.

Reply:
@cfergeau we need all tags -- unless I'm missing something fundamental. Sadly, if you specify that you want the tagged items to run, only those will run. So in particular, if you're running a `linux` tag and some part is not tagged as `linux`, it won't run. It clutters the feature files, but using "nonlinux", "nonwindows" and "nondarwin" tags (then negating them at the call when you want linux, windows, or darwin) does not improve situation much and it's confusing. So I left it as it is.

Reply:
Thanks for checking :)